gulpfile.js

```diff
@@ -658,68 +701,112 @@ function formattingCheckSpell(data, pack, file, options = { checkLinks: true })
 	}
 
 	//Check spell description for unlinked references to conditions
-	if (options.checkLinks) {
+	if (options.checkLinks == true) {
 		let description = data.data.description.value
-		let result = searchDescriptionForUnlinkedCondition(description);
-		if (result.found) {
-			addWarningForPack(`${file}: Found reference to ${result.match} in description without link".`, pack);
-		}
+
+        // Check references to conditions
+        let conditionResult = serchDescriptionForUnlinkedReference(description, conditionsRegularExpression);
+		if (conditionResult.found) {
```

Comment:
Check whitespace on this line. The `if` statement should probably line up wit the `let` declaration above it.